index.html

```diff
@@ -407,15 +407,5 @@ <h2>More information</h2>
       </div>
     </div>
     <script src="https://yandex.st/highlightjs/7.4/highlight.min.js"></script>
-    <script type="text/javascript">
-      hljs.initHighlightingOnLoad();
```

Comment:
it looks like we're also removing highlightjs here 🤔

Reply:
whoops 😅